site/docs/v1/tech/shared/_scim-limits.adoc

```diff
@@ -1,20 +1,15 @@
-The current SCIM implementation does not support:
-
-* Bulk operations
+The current SCIM implementation does not support bulk operations.
 
 Bulk API operations are available via link:/docs/v1/tech/apis/[FusionAuth APIs], but are not currently supported via SCIM compatible endpoints. If this functionality is required, please file a https://github.com/fusionauth/fusionauth-issues/issues[GitHub issue] explaining your use case.
 
 The list operation is limited to 10,000 users if you are using the Elasticsearch search engine. This https://github.com/FusionAuth/fusionauth-issues/issues/494[GitHub issue] discusses this limitation.
 
-You cannot create a FusionAuth Registration for a user provisioned with SCIM. If you need to register the user for one or more Applications, you have a few options. You may:
-
-* use the link:/docs/v1/tech/lambdas/scim-user-request-converter[SCIM User Request Converter Lambda].
-* use the link:/docs/v1/tech/events-webhooks/events/user-create-complete[`user.create.complete` webhook].
-
-Either works. The correct choice depends on if you want to do the processing within FusionAuth using a lambda or using an external server.
-
-While it is not enforced, or a hard limitation, it is recommended to configure at most one SCIM server per tenant.
+You cannot create a FusionAuth Registration for a user provisioned with SCIM directly during the provisioning process. See link:/docs/v1/tech/core-concepts/scim#adding-registrations[Adding Registrations] for workarounds.
 
-While you can have multiple SCIM servers per configured within one tenant, at the end of the day you only have one user data store per tenant.
+While it is not enforced, or a hard limitation, it is recommended to configure at most one SCIM server per tenant. While you can have multiple SCIM servers configured to provision users in one FusionAuth tenant, at the end of the day you only have one user data store in each tenant. Conflicts with creation, updates and deletion could happen; they won't be automatically resolved by FusionAuth. If you need to use multiple SCIM servers, ensure your clients can handle such conflicts.
 
-Conflicts with creation, updates and deletion could happen; they won't be automatically resolved by FusionAuth. If you need to use multiple SCIM servers, ensure your clients can handle such conflicts.
+Okay, okay, you want multiple SCIM clients to provision users into a single FusionAuth tenant. Let's talk about the complexities. Imagine you have an Okta SCIM client provisioning users into a FusionAuth tenant and an Azure AD SCIM client provisioning users into the same tenant. Issues with this approach include:
```

Comment:
The list doesn't format correctly
![image](https://github.com/FusionAuth/fusionauth-site/assets/3409780/88e9d622-f14d-458e-af48-746ef7d14cb7)

In my experience something like this works:

```
Lead in paragraph:
+
 * Bullet 1
 * Bullet 2
 * Bullet 3
```

An extra line before the list starts and put a space before each bullet point.